configs/yolox/hyp.scratch.yaml

```diff
@@ -0,0 +1,45 @@
+optimizer:  # TODO align with yolox official
+  optimizer: momentum
+  momentum: 0.9  # SGD momentum/Adam beta1
+  nesterov: True # update gradients with NAG(Nesterov Accelerated Gradient) algorithm
+  loss_scale: 1.0 # loss scale for optimizer
+  group_param: filter_bias_and_bn # group param strategy
+  gp_weight_decay: 0.0005  # group param weight decay 5e-4
+
+loss:
+   name: YOLOxLoss
+   input_size: [640, 640]
+
+#
+#ms_mode: 1
+#ms_jit: False
+
+conf_thres: 0.001
+iou_thres: 0.65
+
+per_batch_size: 8  # 8 * 8 = 64
+img_size: 640
+sync_bn: False
+ms_amp_level: O0
+keep_loss_fp32: True
+
+data:
+  num_parallel_workers: 4
+
+  train_transforms:
+    - {func_name: mosaic, prob: 1.0, mosaic9_prob: 0.0, translate: 0.1, scale: 0.9, degrees: 10.0, shear: 2.0}
+    - {func_name: mixup, prob: 1.0, alpha: 8.0, beta: 8.0, needed_mosaic: True } # TODO chage to yolox mixup
```

Comment:
Is there anything different from yolox mixup？

Reply:
There are some slignt difference. The hsv_gain in the original version is a biad that adds to the img value, while it is multipying factor in mindyolo. Also, there is a random seletion button determins whether each of h, s, v channel are open or not.